.gitignore

```diff
@@ -65,3 +65,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
 fastlane/test_output
+
+# new things bla bla bla
```

Comment:
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 

- Duis aute irure dolor in reprehenderit 
- in voluptate velit esse cillum dolore 
- eu fugiat nulla pariatur

> Excepteur sint occaecat cupidatat 

[non proident](http://google.com) sunt in **culpa** qui _officia_ deserunt ~~mollit~~ anim id est laborum.

1. one
2. two
3. three

Reply:
Testing a comment

Reply:
Testing